membership-attribute-service/test/controllers/AttributeControllerTest.scala

```diff
@@ -440,19 +456,20 @@ class AttributeControllerTest extends Specification with AfterAll with Idiomatic
 
       status(result) shouldEqual OK
       val jsonBody = contentAsJson(result)
-      System.out.println(jsonBody)
+      println(Json.prettyPrint(jsonBody))
```

Comment:
specs2 doesn't seem to have a `withClue` (or `info`) method so we have to print it every time?